_includes/cloud-db/cloud-summary-db-version.md

```diff
@@ -1 +1 @@
-A database version represents the FeatureBase version the database runs on.
\ No newline at end of file
+A database version represents the FeatureBase version the database runs on. It is denoted as a "release number"-"commit hash" that corresponds to a commit to the [open source repository](https://github.com/FeatureBaseDB/featurebase). When you create a new database in FeatureBase Cloud, it will use the latest version. 
```

Comment:
There's 3 pieces of information here, two of which are saying the same thing:
* FB database version = fb version at time of creation
* Release number commit hash on the open source 
* FB db when created uses the latest version (of the system)

Also, is there a specific reason the commit/hash version is relevant here other than general information? My vote would be to leave that out, because I'm unsure how many users will dive into the GitHub commits to see what version of the application they're dealing with?

Suggested rewrite:
```
The FeatureBase application version number is assigned to each Database when it is created. The Database version then remains static after FeatureBase Cloud is automatically upgraded.
```

Reply:
Sorry not doing this one for a couple reasons:
1. this is a version they see, so it should be explained. I do believe some will go to the actual commit considering some of our customers do now. They would do this to see how far behind and what functionality is missing. This also helps us internally help troubleshoot, so it's good to have documented.
2. This isn't assigned nor does it remain static. It's the version of featurebase. We don't automatically upgrade, but in the case we do, this version will change